hipify_torch/hipify_python.py

```diff
@@ -708,6 +712,18 @@ def pattern(self):
 RE_THC_GENERIC_FILE = re.compile(r'#define THC_GENERIC_FILE "([^"]+)"')
 RE_CU_SUFFIX = re.compile(r'\.cu\b')  # be careful not to pick up .cuh
 
+#  This function takes in the custom json file and looks for custom_mappings available.
+def update_custom_mappings():
+    if os.path.exists(custom_mapping_file):
+        with open(custom_mapping_file, 'r') as f:
+            json_data = json.load(f)
+        custom_mapping_list = json_data['custom_map']
+        for custom_map in custom_mapping_list:
+            src = list(custom_map.keys())[0]
```

Comment:
I think we should just be able to use key/value pairs from `custom_map` to put into `CUSTOM_TRIE` and `CUSTOM_SPECIAL_MAP[src]`

Reply:
Done